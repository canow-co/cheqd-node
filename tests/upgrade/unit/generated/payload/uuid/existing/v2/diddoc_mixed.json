{
    "didDoc": {
        "context": [
            "abc",
            "def"
        ],
        "id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
        "controller": [
            "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d"
        ],
        "verificationMethod": [
            {
                "id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-1",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
                "verificationMaterial": "z6Mkhu3KZFd39BSD8kDNbiWyXsXXZ9uy8qMNrqbrkkV9z4YD"
            },
            {
                "id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-2",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
                "verificationMaterial": "z6MkwUMt89ooTRMGDrPXX1Uei93ouEhuNpgcXtRaazE33Dcw"
            },
            {
                "id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-3",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
                "verificationMaterial": "z6MksenQTTP7bs4HxfAisuJbh6qxW9ffucX5p5tpPkeQ4kmm"
            },
            {
                "id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-4",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
                "verificationMaterial": "z6MkvLgz1avSwuTpMn9KwHR9edSrkJWypozqJgcLkRc3Ahn5"
            }
        ],
        "authentication": [
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-1"
            },
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-2"
            }
        ],
        "assertionMethod": [
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-3"
            }
        ],
        "capabilityInvocation": [
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-4"
            },
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-1"
            }
        ],
        "capabilityDelegation": [
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-4"
            },
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-2"
            }
        ],
        "keyAgreement": [
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-1"
            },
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-2"
            },
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-3"
            },
            {
                "verification_method_id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-4"
            }
        ],
        "service": [
            {
                "id": "did:canow:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#service-1",
                "serviceType": "type-1",
                "serviceEndpoint": [
                    "endpoint-1"
                ]
            }
        ],
        "alsoKnownAs": [
            "alias-1",
            "alias-2"
        ]
    },
    "metadata": {
        "created": "2021-01-01T00:00:00Z",
        "updated": "2021-01-01T00:00:00Z",
        "deactivated": false,
        "versionId": "80e5fa13-d60f-49fc-97b3-f75201cd7dde",
        "nextVersionId": "",
        "previousVersionId": ""
    }
}