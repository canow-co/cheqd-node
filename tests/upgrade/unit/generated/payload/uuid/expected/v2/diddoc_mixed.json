{
    "didDoc": {
        "context": [
            "abc",
            "def"
        ],
        "id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
        "controller": [
            "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884"
        ],
        "verificationMethod": [
            {
                "id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-1",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
                "verificationMaterial": "z6Mkhu3KZFd39BSD8kDNbiWyXsXXZ9uy8qMNrqbrkkV9z4YD"
            },
            {
                "id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-2",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
                "verificationMaterial": "z6MkwUMt89ooTRMGDrPXX1Uei93ouEhuNpgcXtRaazE33Dcw"
            },
            {
                "id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-3",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
                "verificationMaterial": "z6MksenQTTP7bs4HxfAisuJbh6qxW9ffucX5p5tpPkeQ4kmm"
            },
            {
                "id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-4",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
                "verificationMaterial": "z6MkvLgz1avSwuTpMn9KwHR9edSrkJWypozqJgcLkRc3Ahn5"
            }
        ],
        "authentication": [
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-1"
            },
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-2"
            }
        ],
        "assertionMethod": [
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-3"
            }
        ],
        "capabilityInvocation": [
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-4"
            },
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-1"
            }
        ],
        "capabilityDelegation": [
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-4"
            },
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-2"
            }
        ],
        "keyAgreement": [
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-1"
            },
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-2"
            },
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-3"
            },
            {
                "verification_method_id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-4"
            }
        ],
        "service": [
            {
                "id": "did:canow:testnet:a35d7486-1749-55bf-96d4-683958a9a884#service-1",
                "serviceType": "type-1",
                "serviceEndpoint": [
                    "endpoint-1"
                ]
            }
        ],
        "alsoKnownAs": [
            "alias-1",
            "alias-2"
        ]
    },
    "metadata": {
        "created": "2021-01-01T00:00:00Z",
        "updated": "2021-01-01T00:00:00Z",
        "deactivated": false,
        "versionId": "80e5fa13-d60f-49fc-97b3-f75201cd7dde",
        "nextVersionId": "",
        "previousVersionId": ""
    }
}